modules/xfem/include/base/XFEMCutElem.h

```diff
@@ -28,26 +28,33 @@ class EFAElement;
 class XFEMCutElem
 {
 public:
-  XFEMCutElem(Elem * elem, unsigned int n_qpoints);
+  XFEMCutElem(Elem * elem, unsigned int n_qpoints, unsigned int n_sides);
   virtual ~XFEMCutElem();
 
 protected:
   unsigned int _n_nodes;
   unsigned int _n_qpoints;
+  unsigned int _n_sides;
   std::vector<Node *> _nodes;
   std::vector<Point> _qp_points;
   std::vector<Real> _qp_weights;
   Real _elem_volume;
+  std::vector<Real> _elem_side_area;
   Real _physical_volfrac;
+  std::vector<Real> _physical_areafrac;
   bool _have_weights;
+  std::vector<bool> _have_face_weights;
   std::vector<Real> _new_weights; // quadrature weights from moment fitting
+  std::vector<std::vector<Real>> _new_face_weights;
```

Comment:
No new data members have doco. Actually, there is no doco at all! And the style of doco is `///` on the line before the member variables, not `//...` after the variable.